src/lookup/tagColorLookup.ts

```diff
@@ -0,0 +1,37 @@
+type TagColorDict = {
+    [key: string]: string;
+};
+
+const lookupTagColor: TagColorDict = {
+    "1": "#ffeb3b",
+    "2": "#cddc39",
+    "3": "#607d8b",
+    "4": "#483562",
+    "5": "#000",
+    "6": "#ffb74d",
+}
+
+const lookupTextColor: TagColorDict = {
+    "1": "#000",
+    "2": "#000",
+    "3": "#fff",
+    "4": "#fff",
+    "5": "#fff",
+    "6": "#000",
+}
+
+export function tagColorLookup(key: string): string {
+    if (key in lookupTagColor) {
+        return lookupTagColor[key];
+    }
+
+    return key;
```

Comment:
What do you think about a wrap-around to ensure that the index will be valid?
```suggestion
    if (key < 0) {
        // throw error or simply return lookupTagColor[0]? Or some special "error" color?
    }
    return lookupTagColor[key % lookupTagColor.length];
```

Reply:
why you using the modulo there?
Than the color will "wrap around" right?

Wouldn't it be better to just throw an error?
Because the color code is not the expected than.

Reply:
Ah okay, that was a little mis-assumption on my end. In the admin dashboard, when we create a new tag, we will select a color there (based on the available colors of the frontend). So yes, you're right, there should never be an invalid color index (unless we reduce the number of indices later on, but that would not make any sense).
Okay, then you can just throw an error, I guess (instead of returning the key if it cannot be found).